data/entity/npc/monster/draconic/dragon.npcs.toml

```
@@ -0,0 +1,132 @@
+[king_black_dragon]
+id = 50
+hitpoints = 2400
+att = 240
+str = 240
+def = 240
+mage = 240
+style = "stab"
+attack_range = 8
+attack_radius = 8
+max_hit_melee = 250
+max_hit_dragonfire = 650
+hunt_mode = "aggressive"
+allowed_under = true
+slayer_xp = 258.0
+categories = ["dragon", "boss"]
+aka = ["kbd"]
+respawn_delay = 16
+examine = "One of the biggest, meanest dragons around."
+
+[bronze_dragon]
+id = 1590
+hitpoints = 1520
+att = 112
+str = 112
+def = 112
+mage = 100
+style = "slash"
+attack_range = 8
+attack_radius = 8
+wander_radius = 4
+max_hit_melee = 120
+max_hit_dragonfire = 500
+hunt_mode = "aggressive"
+allowed_under = true
+slayer_xp = 125.0
+categories = ["dragon"]
+respawn_delay = 30
+examine = "Its scales seem to be made of bronze."
+
+[iron_dragon]
+id = 1591
+hitpoints = 1890
+att = 165
+str = 165
+def = 165
+mage = 100
+style = "slash"
+attack_range = 8
+attack_radius = 8
+wander_radius = 2
+max_hit_melee = 170
+max_hit_dragonfire = 500
+hunt_mode = "aggressive"
+allowed_under = true
+slayer_xp = 173.2
+categories = ["dragon"]
```

```suggestion
combat_anims = "dragon"
combat_sounds = "dragon"
categories = ["dragons"]
```

Categories is now really only used for the slayer category